src/witan/gwyn/schemas.clj

```diff
@@ -31,7 +32,7 @@
 
 (def HistoricalFireRiskScores
   (make-ordered-ds-schema [[:address s/Str] [:name s/Str] [:risk-score java.lang.Double]
-                           [:date-last-risk-assessed s/Any]]))
+                           [:date-last-risk-assessed sc/ISO-Date-Time]]))
```

Comment:
Are we confident about the human readability of ISO dates? We definitely don't need times. Dates will be input by hand by users into a csv